prereise/gather/data/BA_County_map.json

```diff
@@ -0,0 +1 @@
ashington__OR", "Yamhill__OR"], "ERCOT": ["Anderson__TX", "Andrews__TX", "Angelina__TX", "Aransas__TX", "Archer__TX", "Armstrong__TX", "Atascosa__TX", "Austin__TX", "Bandera__TX", "Bastrop__TX", "Baylor__TX", "Bee__TX", "Bell__TX", "Bexar__TX", "Blanco__TX", "Borden__TX", "Bosque__TX", "Brazoria__TX", "Brazos__TX", "Brewster__TX", "Briscoe__TX", "Brooks__TX", "Brown__TX", "Burleson__TX", "Burnet__TX", "Caldwell__TX", "Calhoun__TX", "Callahan__TX", "Cameron__TX", "Carson__TX", "Castro__TX", "Chambers__TX", "Cherokee__TX", "Childress__TX", "Clay__TX", "Coke__TX", "Coleman__TX", "Collin__TX", "Collingsworth__TX", "Colorado__TX", "Comal__TX", "Comanche__TX", "Concho__TX", "Cooke__TX", "Coryell__TX", "Cottle__TX", "Crane__TX", "Crockett__TX", "Crosby__TX", "Dallas__TX", "Dawson__TX", "DeWitt__TX", "Deaf Smith__TX", "Delta__TX", "Denton__TX", "Dickens__TX", "Dimmit__TX", "Duval__TX", "Eastland__TX", "Ector__TX", "Edwards__TX", "Ellis__TX", "Erath__TX", "Falls__TX", "Fayette__TX", "Fisher__TX", "Floyd__TX", "Foard__TX", "Fort Bend__TX", "Franklin__TX", "Freestone__TX", "Frio__TX", "Galveston__TX", "Garza__TX", "Gillespie__TX", "Glasscock__TX", "Goliad__TX", "Gonzales__TX", "Grayson__TX", "Grimes__TX", "Guadalupe__TX", "Hall__TX", "Hamilton__TX", "Hardeman__TX", "Harris__TX", "Haskell__TX", "Hays__TX", "Henderson__TX", "Hidalgo__TX", "Hill__TX", "Hood__TX", "Hopkins__TX", "Houston__TX", "Howard__TX", "Hunt__TX", "Irion__TX", "Jack__TX", "Jackson__TX", "Jeff Davis__TX", "Jim Hogg__TX", "Jim Wells__TX", "Johnson__TX", "Jones__TX", "Karnes__TX", "Kaufman__TX", "Kendall__TX", "Kenedy__TX", "Kent__TX", "Kerr__TX", "Kimble__TX", "King__TX", "Kinney__TX", "Kleberg__TX", "Knox__TX", "La Salle__TX", "Lamar__TX", "Lampasas__TX", "Lavaca__TX", "Lee__TX", "Leon__TX", "Limestone__TX", "Live Oak__TX", "Llano__TX", "Loving__TX", "McCulloch__TX", "McLennan__TX", "McMullen__TX", "Madison__TX", "Martin__TX", "Mason__TX", "Matagorda__TX", "Maverick__TX", "Medina__TX", "Menard__TX", "Midland__TX", "Milam__TX", "Mills__TX", "Mitchell__TX", "Motley__TX", "Nacogdoches__TX", "Navarro__TX", "Nolan__TX", "Nueces__TX", "Ochiltree__TX", "Oldham__TX", "Palo Pinto__TX", "Parker__TX", "Pecos__TX", "Presidio__TX", "Rains__TX", "Reagan__TX", "Real__TX", "Reeves__TX", "Refugio__TX", "Roberts__TX", "Robertson__TX", "Rockwall__TX", "Runnels__TX", "Rusk__TX", "San Patricio__TX", "San Saba__TX", "Schleicher__TX", "Scurry__TX", "Shackelford__TX", "Somervell__TX", "Starr__TX", "Stephens__TX", "Sterling__TX", "Stonewall__TX", "Sutton__TX", "Swisher__TX", "Tarrant__TX", "Taylor__TX", "Terrell__TX", "Throckmorton__TX", "Titus__TX", "Tom Green__TX", "Travis__TX", "Upton__TX", "Uvalde__TX", "Val Verde__TX", "Van Zandt__TX", "Victoria__TX", "Waller__TX", "Ward__TX", "Washington__TX", "Webb__TX", "Wharton__TX", "Wheeler__TX", "Wichita__TX", "Wilbarger__TX", "Willacy__TX", "Williamson__TX", "Wilson__TX", "Winkler__TX", "Wise__TX", "Young__TX", "Zapata__TX", "Zavala__TX"], "SCL_TPWR": ["Thurston__WA"]}
```

Comment:
Do we want to format this as if it were a python dict?

Reply:
It is a python dict if I remember correctly. You are suggesting we put it in a `const.py` as a dict explicitly?

Reply:
`BA_County_map.json`. That's why the linter does not complain.

Reply:
Yes. The linter won't go through a `json` file.

Reply:
It could be in a `const.py`, or a `ba_county_map.py`, or in `BA_County_map.json` but formatted better.

Unrelated, but what is the source of this data? Do we need to go through the data intake procedure?

Reply:
I generated this `json` file simply by dumping a python `dict` to it. Details see `Dropbox (Gates Ventures)/BreakthroughEnergySciences/Explorations/BainanX/external connection requests/County to BA map`
If we go with `json`, how should we reformat it.

This is actually the output of my script + manual process when addressing an external request last year, again details could be found in the file directory above. Short answer, the raw data can't be redistributed and we use the data we generated here, thus no data intake need to be conducted

Reply:
`black` can be used to manually reformat this file as if it were python, and it should still be valid JSON I think.

Reply:
That's good to know. Will see how it works.

Reply:
There is an issue for using black to reformat json file: 
![image](https://user-images.githubusercontent.com/52716585/124310234-80758d00-db31-11eb-8ef7-5c9c47acffe0.png)

Reply:
Update: it works after I manually fix the issue above.